API.md

```diff
@@ -746,6 +747,16 @@ var func = Joi.func();
 func.validate(function () {}, function (err, value) { });
 ```
 
+#### `func.arity(n)`
+
+Specifies the arity of the function where:
+- `n` - the arity expected.
+
+```javascript
+var schema = Joi.func().arity(2);
```

Comment:
noted: 
I copy pasted from number.min, 
migt be then interesting to go over all the api.md to replace all the var. 
(in a dedicated PR of course)